src/views/onboarding/collection-view/index.tsx

```diff
@@ -53,6 +53,7 @@ export default function OnboardingCollectionView({
 					isOrdered={ordered}
 					tutorials={tutorials.map((t: ClientTutorialLite) => ({
 						id: t.id,
+						collectionId: id,
```

Comment:
`collectionId` is now needed for `TutorialCard` (or rather `TutorialCardWithAuth` as used in `TutorialCardGrid`), in order to display progress.